### ISSUE TITLE
class not found error from localstack server less

### ISSUE THREAD

**Issue by oh-isuri-bandara** [localstack/localstack#1802]
Hi, I deployed my maven function and then invoked as below. But when I try to run docker image I get ClassNot Found error.

```
C02TX00EHV2H:hellome xx$ serverless invoke local  -f hellome --log
Serverless: Packaging service...
Serverless: Building Docker image...
START RequestId: fc013012-d402-1cf2-ca95-2ec1ab9f45ab Version: $LATEST

2019-11-25 02:30:59 fc013012-d402-1cf2-ca95-2ec1ab9f45ab INFO  Handler:18 - received: null



END RequestId: fc013012-d402-1cf2-ca95-2ec1ab9f45ab
REPORT RequestId: fc013012-d402-1cf2-ca95-2ec1ab9f45ab	Init Duration: 5309.67 ms	Duration: 3192.98 ms	Billed Duration: 3200 ms	Memory Size: 1536 MB	Max Memory Used: 86 MB	


{"statusCode":200,"body":"{\"message\":\"Go Serverless v1.x! Your function executed successfully!\",\"input\":null}","headers":{"X-Powered-By":"AWS Lambda \u0026 serverless"},"isBase64Encoded":false}

C02TX00EHV2H:hellome xx$ docker images
REPOSITORY                                              TAG                 IMAGE ID            CREATED             SIZE
lambci/lambda                                           java11              b4f3365c08ef        28 hours ago        395MB

C02TX00EHV2H:hellome xx$ docker run -v "$PWD":/var/task lambci/lambda:java11 com.serverless.Handler'{"resource": "testme", "httpMethod": "GET"}'
START RequestId: 86d2fab4-3805-1d43-d11a-6ec85b3c4347 Version: $LATEST
Class not found: com.serverless.Handler{"resource": "testme", "httpMethod": "GET"}: java.lang.ClassNotFoundException
java.lang.ClassNotFoundException: com.serverless.Handler{"resource": "testme", "httpMethod": "GET"}
	at java.base/java.net.URLClassLoader.findClass(Unknown Source)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Unknown Source)
```

Below is my Handler class : (I used Maven, Java 11 runtime)
```
package com.serverless;

import com.amazonaws.services.lambda.runtime.Context;
import com.amazonaws.services.lambda.runtime.RequestHandler;
import java.util.Collections;
import java.util.Map;
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

public class Handler implements RequestHandler<Map<String, Object>, ApiGatewayResponse> {
  private static final Logger LOG = LogManager.getLogger(Handler.class);

  public Handler() {
  }

  public ApiGatewayResponse handleRequest(Map<String, Object> input, Context context) {
    LOG.info("received: {}", input);
    Response responseBody = new Response("Go Serverless v1.x! Your function executed successfully!", input);
    return ApiGatewayResponse.builder().setStatusCode(200).setObjectBody(responseBody).setHeaders(Collections.singletonMap("X-Powered-By", "AWS Lambda & serverless")).build();
  }
}
```

**Comment by whummer** [comment]
Thanks for reporting @oh-isuri-bandara . Instead of running the `docker run ...` command directly, can you try to deploy and invoke the Lambda function in LocalStack, similar to what you would do against real AWS:
```
$ pip install awscli-local
$ awslocal lambda create-function ...
$ awslocal lambda invoke ...
```

**Comment by oh-isuri-bandara** [comment]
Thanks @whummer 

Is serverless supporting java11 runtime yet ?

I tried your suggestion and below is what happened. 

$ awslocal lambda create-function --environment Variables={functionStage=local} --region us-east-1 --function-namehellome --runtime java11 --handler com.serverless.Handler --memory-size 128 --role arn:aws:iam::123456:role/irrelevant --zip-file fileb://target/hellome-dev.jar

**An error occurred (ValidationError) when calling the CreateFunction operation: Unable to find handler script (/tmp/localstack/zipfile.32f885d6/com/serverless.py) in Lambda archive. Config for local mount, docker, remote: "False", "True", "True"**

but it worked fine for java8 runtime. 

$ awslocal lambda create-function --environment Variables={functionStage=local} --region us-east-1 --function-nae hellome --runtime java8 --handler com.serverless.Handler --memory-size 128 --role arn:aws:iam::123456:role/irrelevant --zip-file fileb://target/hellome-dev.jar
{
    "FunctionName": "hellome", 
    "LastModified": "2019-11-25T21:06:34.753+0000", 
    "RevisionId": "426287b6-288e-49e6-bfd0-dc90c623b70e", 
    "MemorySize": 128, 
    "Environment": {
        "Variables": {
            "functionStage": "local"
        }
    },

**Comment by oh-isuri-bandara** [comment]
any update on this ? Just wants to know when localstack is supporting java 11 runtime...

**Comment by whummer** [comment]
Thanks for reporting @oh-isuri-bandara . This should be fixed in #1963 - please report here if the problem persists. Thanks
